improver_tests/snow_fraction/test_SnowFraction.py

```diff
@@ -151,6 +177,18 @@ def test_input_name_matches_both_phases_error():
         SnowFraction()(iris.cube.CubeList([rain, snow]))
 
 
+def test_coercing_units():
+    """Test the process function with input cubes of different but compatible units"""
+    rain, snow = setup_cubes()
+    rain.convert_units("mm h-1")
+    expected_data = np.array([[1.0, 1.0], [2.0 / 3.0, 0.0]], dtype=np.float32)
```

Comment:
This occurs in a couple of places - does it belong in the setup functions / globals?

Reply:
Yes. Moved to be global.